platform/kube-state-metrics/launch.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+	"github.com/sipb/homeworld/platform/keysystem/worldconfig/paths"
+	"github.com/sipb/homeworld/platform/kubernetes/wrapper"
+	"os"
+	"os/exec"
+	"strconv"
+)
+
+const KubeConfigPath = "/etc/homeworld/config/kubeconfig-kube-state-metrics"
+const MetricsPort = 9104
+const MetaMetricsPort = 9105
+
+func LaunchKSM(kubeconfig string, port, metaport int) error {
+	portStr := strconv.FormatInt(int64(port), 10)
+	metaportStr := strconv.FormatInt(int64(metaport), 10)
```

Comment:
port numbers are u16

Reply:
FormatInt takes an int64, so I have to convert to this.